### PR TITLE
Login  triggered when pressing enter #6322

### LINKED ISSUE #6322: Login should be triggered when pressing enter
###### Reproduction Steps

1. Open the login screen
2. Type in username and password (using a connected keyboard - mine was my bluetooth keyboard on the emulator)
3. Press Enter

###### Expected Result

4. Login should occur

###### Actual Result

4. Login button is highlighted

###### Debug Info

Tested on the API 16 emulator, 2.11alpha10

###### Research
- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by coma123**
## Description
When enter key pressed after writing password, login is triggered

## Fixes #6322 

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java [hunk 1]
```diff
@@ -151,6 +151,22 @@ private void initAllContentViews() {
         mUsername = mLoginToMyAccountView.findViewById(R.id.username);
         mPassword = mLoginToMyAccountView.findViewById(R.id.password);
 
+        mPassword.setOnKeyListener((v, keyCode, event) -> {
+            if (event.getAction() == KeyEvent.ACTION_DOWN)
+            {
+                switch (keyCode)
+                {
+                    case KeyEvent.KEYCODE_DPAD_CENTER:
+                    case KeyEvent.KEYCODE_ENTER:
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java:160]:
```suggestion
                    case KeyEvent.KEYCODE_ENTER:
                    case KeyEvent.KEYCODE_NUMPAD_ENTER:
```

File: AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java [hunk 2]
```diff
@@ -151,6 +151,22 @@ private void initAllContentViews() {
         mUsername = mLoginToMyAccountView.findViewById(R.id.username);
         mPassword = mLoginToMyAccountView.findViewById(R.id.password);
 
+        mPassword.setOnKeyListener((v, keyCode, event) -> {
+            if (event.getAction() == KeyEvent.ACTION_DOWN)
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java:155]: Just to note: our coding conventions would prefer One True Brace style:

https://github.com/ankidroid/Anki-Android/blob/edcdfe0b44d4c1359ebe25b17043203704201e11/docs/code_conventions/AnkiDroid_code_conventions.html#L1122-L1140

**Comment by david-allison-1** [comment]
P.S: Thanks so much for the code style fixes 💯

**Comment by coma123** [comment]
@david-allison-1 thanks for suggestions, all of them are done as I think 😄

**Comment by mikehardy** [comment]
Nice change @coma123 ! I notice this all the time in testing (I'm always reinstalling the app and logging in again) so you have one happy user already :smile:

**Comment by coma123** [comment]
@mikehardy 😄 thanks

**Comment by mikehardy** [comment]
Hello! with the 2.14 release we started accepting donations with the stated goal of accepting money from users (the users' way of saying "Thanks!") to support maintenance and development of AnkiDroid.

We are looking through PRs merged in releases going back to release 2.10 (where we have good record keeping of what's in releases) to let our contributors know.

If you'd like to submit an expense related to your work here, to share in a portion of our hopefully-sustainable monthly disbursement, please read here and do so in the next couple of days:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process

This is our first time handling donations and attempting to compensate anyone with these processes, so please be flexible and patient with us

Thank you for your contribution to AnkiDroid!

**Pull request merged**